src/settings/VoucherNumber/SettingsVoucherNumber.js

```diff
@@ -20,27 +23,41 @@ import SettingsVoucherNumberForm from './SettingsVoucherNumberForm';
 class SettingsVoucherNumber extends Component {
   static manifest = Object.freeze({
     voucherNumber: VOUCHER_NUMBER_START,
+    sequenceNumber: { initialValue: DEFAULT_VOUCHER_START_NUMBER },
```

Comment:
where it is used?

Reply:
It used when start number field not edited. Because by default `%{sequenceNumber}` return object and I can't create correct path for post

Reply:
this makes no sense to me. initialValue isn't referenced in the code anywhere. How it's supposed to work? What I see in the code - it's used as a plain value, not the object.

Reply:
For local resource we can set initialValue. And we use this construction in application(https://github.com/folio-org/ui-invoice/blob/master/src/invoices/InvoicesList/InvoicesList.js#L100). I also looked at the strip-connect to make sure. I don't know why this don't explained in doc

Reply:
But why do we need to do so in this very case?